Source/ACE/Managers/GuidManager.cs

```diff
@@ -164,5 +169,23 @@ public static ObjectGuid NewItemGuid()
         {
             return new ObjectGuid(itemAlloc.Alloc());
         }
+
+        /// <summary>
+        /// Returns Most Recently assigned Guid for Items / Player Items, to be used only for informational purposes.
+        /// </summary>
+        /// <returns></returns>
+        public static ObjectGuid CurrentItemGuid()
```

Comment:
Doing the +1 to GuidManager.Current was wrong as that is mislabled, it is +1'd on Alloc after passing the current value. Fixed my incorrect math and renamed functions to fall into line with what they are doing.